### PR TITLE
chore(build): add full dockerfile containing yarn step

### PR DESCRIPTION
**Pull request by typecastcloud**
## Description

Full build dockerfile for frontend registration image with yarn stage.

## Why

We can create images without yarn installed and this change streamlines dockerfiles across portal repositories.

## Issue

Refs: #186

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: .conf/Dockerfile.full [hunk 1]
```diff
@@ -0,0 +1,49 @@
+###############################################################
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
```
Comment (oyo) [inline .conf/Dockerfile.full:2]: for new files please specify current year

**Review by oyo — APPROVED** [review]
please update year in license header - otherwise looks good
@evegufy please also have a look

File: .conf/Dockerfile.full (hunk 1 above)
Comment (typecastcloud) [reply .conf/Dockerfile.full]: Done

**Comment by typecastcloud** [comment]
@evegufy @oyo any update on this?

**Review by evegufy — CHANGES_REQUESTED** [review]
sorry for the late review, looks good in general.

I'd just change the base branch to dev as we still have that the dev flow in place for this repo.

And another thing: could you please rename the prebuilt dockerfile to `Dockerfile.prebuilt` and adjust the referencing in the workflow files accordingly, [example](https://github.com/Cofinity-X/entry-portal-frontend-registration/blob/feat/add-full-build-dockerfile/.github/workflows/build.yml#L90), so that we improve also on the understanding of why there are now two dockerfiles.

Thank you!

**Comment by typecastcloud** [comment]
> sorry for the late review, looks good in general.
> 
> I'd just change the base branch to dev as we still have that the dev flow in place for this repo.
> 
> And another thing: could you please rename the prebuilt dockerfile to `Dockerfile.prebuilt` and adjust the referencing in the workflow files accordingly, [example](https://github.com/Cofinity-X/entry-portal-frontend-registration/blob/feat/add-full-build-dockerfile/.github/workflows/build.yml#L90), so that we improve also on the understanding of why there are now two dockerfiles.
> 
> Thank you!

Hi Evelyn,

Thanks for the reply. At first I did not want to touch the existing Dockerfile since external workflows/jobs might be using the previous name.

I can't fully evaluate the risks but made the requested changes.

Cheers

**Review by evegufy — APPROVED** [review]

**Pull request merged**